spec.html

```diff
@@ -21638,16 +21637,30 @@
           </emu-note>
         </emu-clause>
 
+        <emu-clause id="sec-resolutionfailure" aoid="ResolutionFailure">
+          <h1>Runtime Semantics: ResolutionFailure( _module_, _resolveMode_ )</h1>
+          <p>The ResolutionFailure abstract operation performs the following steps:</p>
+          <emu-alg>
+            1. Assert: _module_.[[Status]] is `"uninstantiated"` or `"instantiating"`.
+            1. If _resolveMode_ is `"namespace"` or `"star"`: return *undefined*.
+            1. Assert: _resolveMode_ is `"default"`.
```

Comment:
Probably should rename "default" to "normal" to avoid confusion with default _export_.